.gitignore

```diff
@@ -30,3 +30,4 @@ changelog.txt
 /patches_old_bak/
 *.py
 *.7z
+/.metadata/
```

Comment:
What generates this folder? If it is from an unusual utility you may want to add it to your own global git ignore instead of including it here.

Reply:
Weird. I don't usually do anything to gitignore files, so not sure how that ended up here. But easy to move into the global.